Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/PgSql/Data/Security/1010-TPDM-ResourceClaims.sql

```diff
@@ -1710,6 +1710,34 @@ BEGIN
     claim_id_stack := array_append(claim_id_stack, claim_id);
 
     -- Processing children of http://ed-fi.org/ods/identity/claims/domains/tpdm/students
+	----------------------------------------------------------------------------------------------------------------------------
```

Comment:
Same thing here, should we also regenerate the file and remove the old resources?

Reply:
ODS-5116 will address the removal, I was trying to avoid duplicate work if TPDM team can provide the claim setup according to their needs.